### PR TITLE
support HTML-mode in Pushover node #467

### PR DESCRIPTION
**Pull request by cowchimp**
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

Adds an option to toggle HTML mode as discussed in #467.
Tested it with open on\off.
Is backward compatible (HTML mode is off)

## Checklist

- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red-nodes/blob/master/CONTRIBUTING.md)
- [x] For non-bugfix PRs, I have discussed this change on the mailing list/slack team.
- [x] I have run `grunt` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

**Comment by cowchimp** [comment]
Hi.
Build now passes after merging upstream into my fork.
I made the changes as we discussed in the issue.
Would you like to change anything else or is this okay to merge?
Thanks.

File: social/pushover/57-pushover.js [hunk 1]
```diff
@@ -35,6 +36,7 @@ module.exports = function(RED) {
             var sound = this.sound || msg.sound || null;
             var url = this.url || msg.url || null;
             var url_title = this.url_title || msg.url_title || null;
+            var html = this.html || msg.html || false;
```
Comment (dceejay) [inline social/pushover/57-pushover.js]: If the user sets the flag to false then it should be honoured. This won't do that - as the this.html will evaluate false and then skip to the next option (msg.html)... so msg.html could always override it.

Comment (cowchimp) [reply social/pushover/57-pushover.js]: Good point. I removed the option to toggle HTML mode from the incoming msg.
Now that I've given it some more thought, I can't think of a reasonable case where you would need to make this dynamic and change based on the `msg` anyway.

**Comment by dceejay** [comment]
Thanks for the PR

**Comment by cowchimp** [comment]
@dceejay `node-red-nodes` depends on `pushover-notifications: ~0.2.3`.
Since that version `pushover-notifications` has released only two version: `0.2.4` and `1.0.0`.
I tested with `0.2.4`.
As for what changed in `1.0.0`, I didn't find a proper changelog at
https://github.com/qbit/node-pushover
but judging from the commit it looks like it was just a caution step because they applied a mass automated reformating to the codebase (using the `standardjs` formatter), and not an API change by choice.
https://github.com/qbit/node-pushover/commit/eeb6d2a118c9c5513d20096fc4232acb5adcf662?w=0#diff-04c6e90faac2675aa89e2176d2eec7d8

**Comment by dceejay** [comment]
... so....  did you test with 1.0 ? - what is your recommendation ?
(Not used pushover in ages... and don't have time to get deep into it now.. so trusting to you :-)

**Comment by cowchimp** [comment]
I just tested with `1.0.0`.
Technically it does work and it's backward compatible, but in my opinion I see no reason to update.

**Comment by dceejay** [comment]
ok - will go with 0.2.4

**Comment by cowchimp** [comment]
Cheers. Thanks.

**Pull request merged**